MANIFEST.in

```diff
@@ -1,5 +1,7 @@
 graft src
 
 include LICENSE
+exclude pytest.ini
 
+global-exclude test_*.py
```

Comment:
I think that these aren't actually needed. `graft src` above is only going to add the contents of `src/` to the MANIFEST and everything else will be ignored by default.  Can you confirm? If the `check-manifest` is erroring then things can be added to https://github.com/scikit-hep/pylhe/blob/087f6abd0ed2c5e3c201938fbcc155650d5be1e4/pyproject.toml#L23-L30
```suggestion
```

Reply:
Looks good: `check-manifest` quotes the files as 'missing from sdist'!